system-tests/driver-config/jenkins.sh

```diff
@@ -29,7 +29,7 @@ targets:
       PATH: "/usr/local/bin:$PATH"
 
     scripts:
-      proxy: http-server --proxy-secure=false -p 4201 -P \$CLUSTER_URL dist
+      proxy: http-server --proxy-secure=false -p 4201 -P \$CLUSTER_URL ../../dist
```

Comment:
This sounds like a major error source could we use something like `npm dir` to determine the correct path like `$(npm dir)/../../dist` ?